stats/main.go

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Eh, can we name the file anything except main? In my head, main.go suggest a main-method (e.g.: cmd/skogul/main.go). Might just be me, but humor me? :D